### ISSUE TITLE
Link-target to appointments, where message has been attached, does not work

### ISSUE THREAD

**Issue by ignoreigor** [tine20/tine20#7231]
Although @pock69 has closed his issue #7230 , i think I create a new case for the bug that the link, labeled to a message in a yellow band, after attachting the message to an appointment, is not working. 
It leads to a new appointment-dialog, instead of showing the linked object. The behaviour is, as far as I have tested, only wrong when linked to appointments, whereas links to other object-types like contacts are working as expected.

The click-event seems properly, on the first look:
onclick="Tine.Felamimail.MessageFileButton.locationClickHandler('Calendar_Model_Event','b1a1294edbb56bf36d665f7d54e98caff065915b')"

but this opens a dialog with an empty object.

Finally a wish: Update the yellow band within a message, which shows the targets where a message is attached, after deleting the message-attachment from an object.

Thank you for your good work. I am always surprised to discover such new features and about the possibilities that Tine 2.0 offers.

**Comment by pschuele** [comment]
hi Igor, thanks for the report and your kind words :)
I'll try to reproduce the problem!

**Comment by pschuele** [comment]
yup, I can confirm that there is something strange with filing attachments to events. I'll have a closer look soon.
